private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -46,6 +48,7 @@ const (
 	disableSymlinksFlagName     = "disable-symlinks"
 	typeFlagName                = "type"
 	typeDeprecatedFlagName      = "include-types"
+	wasmCompilationCacheDir     = "wasmplugins-compiled"
```

Comment:
Same comment - this shouldn't live as a const in command/*, this should be somewhere else. Perhaps bufcli